packages/lib/src/core/Analytics/utils.ts

```diff
@@ -192,7 +195,15 @@ export const getCardConfigData = (cardProps: CardConfiguration): CardConfigData
         hasOnLoad: onLoad !== CardInputDefaultProps.onLoad,
         // Card level props
         hasOnBinLookup: !!onBinLookup,
-        hasOnEnterKeyPressed: !!onEnterKeyPressed
+        hasOnEnterKeyPressed: !!onEnterKeyPressed,
+        /**
+         * Fastlane
+         */
+        // CHECK: Do we always append the prop even if it is not used? e.g. merchants not using fastlane
```

Comment:
Not sure what is the best here

Reply:
For other, _universal_, props we won't to know what config the merchant has ended up with, be it something they've set or a default that they've picked up.
But for something opt-in like Fastlane, we should only send it if it's relevant, imo